lib/util.js

```diff
@@ -136,6 +137,28 @@ util = {
 
             callback(null, data);
         });
+    },
+
+    /**
+     * Writes a JSON blob to the given path.
+     *  * If ``options.pretty`` is true, output will be pretty printed. (Default false)
+     *  * If ``options.overwrite`` is false, output file will be overwritten (Default true)
+     *
+     * @param path
+     * @param data
+     * @param options
+     * @param callback
+     */
+    writeJSON: function (path, data, options, callback) {
```

Comment:
Well, not a good idea to have an `object` as options parameter for very low level functions... needlessly causes new object construction and need to validate if options param is undefined.